src/components/Table/index.js

```diff
@@ -163,7 +163,7 @@ const Table = ({
 
       <tbody>
         {map(elements, (element, index) => (
-          <React.Fragment>
+          <React.Fragment key={`cell-${index}`}>
```

Comment:
I'm not sure we should rely on this `index`

Reply:
cf https://www.atrakeur.com/blog/web/les-travers-des-react-keys/ ^_^

Reply:
Since I prefixed the key with `cell`, there will never be twice the same key. Plus this is what is already being done above in the code, and by yourself if git history is correct ;)

Reply:
The problem is not about having it twice :) The problem is that if we remove the first element, the index of all the other elements will change and so they'll be rerendered :(.
And I'm not always right ^_^

Reply:
ooooh indeed good point. I'll change this :)

Reply:
Fixed in both cases

Reply:
You da best ! approved